### DISCUSSION TITLE
how do I replace the sampler and scheduler?

### DISCUSSION THREAD

**Discussion by Kanape21** [lllyasviel/Fooocus#3529] [Q&A]
The translation is done by deepl, sorry
how to replace the sampler and Scheduler, which is by default, with your own, in our case you need to install dpmpp_3m_sde_gpu - karras
![image](https://github.com/user-attachments/assets/85d73cf3-83ef-48fd-b2f0-318a1452fea2)

**Comment by Rogala** [comment]
Change the preset file you are working with for Initial settings in config.txt
`Fooocus\presets\`

**Reply by Kanape21** [reply]
I must have expressed myself incorrectly.
"Performance" > "lightning"

**Reply by Rogala** [reply]
The sampler and scheduler are set when Preset is selected, Performance is only the processing speed.
